P1684-mdarray/P1684.md

```diff
@@ -619,47 +635,44 @@ public:
       const mdarray<OtherElementType, OtherExtents, 
                    OtherLayoutPolicy, OtherContainer>& other);
 
-  template<class Alloc, class... SizeTypes>
-    requires(uses_allocator_v<container_type, Alloc>)
+  // [mdarray.ctors.alloc], mdarray constructors with allocators
+  template<class Alloc>
     constexpr mdarray(const Extents& ext, const Alloc& a);
```

Comment:
Why do we have both a version with and without allocator?
Can't we do
```suggestion
  template<class Allocator = typename container_type::allocator_type>
    constexpr mdarray(const Extents& ext, const Allocator& alloc = Allocator());
```

Reply:
Christian pointed out that it does not work if underlying container is `std::array`